src/ode_solver/rrk_explicit_ode_solver.cpp

```diff
@@ -25,8 +25,8 @@ real RRKExplicitODESolver<dim,real,n_rk_stages,MeshType>::compute_relaxation_par
     for (int i = 0; i < n_rk_stages; ++i){
         for (int j = 0; j < n_rk_stages; ++j){
             real inner_product = compute_inner_product(this->rk_stage[i],this->rk_stage[j]);
-            numerator += this->butcher_tableau->b(i) *this-> butcher_tableau->a(i,j) * inner_product; 
-            denominator += this->butcher_tableau->b(i)*this->butcher_tableau->b(j) * inner_product;
+            numerator += this->butcher_tableau->get_b(i) *this-> butcher_tableau->get_a(i,j) * inner_product; 
```

Comment:
could be made more efficient if you define `const double b_i = this->butcher_tableau->get_b(i)` before  `j` loop; that way you fetch it only n_rk_stages times, instead of fetching it n_rk_stages*n_rk_stages times

Reply:
Good catch. I changed that. After doing a quick scan, I don't see any other places where `get_*` is called enough to merit storing it.